release-notes/ugandaemr-337.md

```diff
@@ -0,0 +1,66 @@
+# 3.3.7 - March 2022 
+
+## New Features
+
+1. Martnernal and Child Health
+  
+   * Antenatal  Services for both Point of Care and Restrospective approach. 
+   * Maternity Services
+   * Postnatal Services
+   * Reports - (105 MCH Section, ANC 105, Marternity 105, 105 PNC )
+   
+2. Safe Male Circumcision
+
+    *Retrospective Data Entry for SMC Services
+    
+3. Create a generic approach to generating  and Sync FHIR Encounters for data exchange
+4. Added Missing Districts
+    
+## Improvements
+ 
+ 1. Add Display Board for Logs
```

Comment:
Where is this

Reply:
Change to 1. Added display board for logs

I think it's the one at Home, system administration, Ugandaemrsync, synctasklogs. If its the one, more explanation will be needed to understand the codes / wording on that board